Makefile

```diff
@@ -3,10 +3,17 @@ TARGET = alacritty
 APP_NAME = Alacritty.app
 ASSETS_DIR = extra
 RELEASE_DIR = target/release
+COMPLETIONS_DIR = $(ASSETS_DIR)/completions
+
+EXTRAS = $(ASSETS_DIR)/alacritty.info $(ASSETS_DIR)/alacritty.man
```

Comment:
Should we maybe precompile these, so they only have to be moved to the right location without requiring any tools to install them?

Reply:
Yeah, that's probably a good idea.

Reply:
I'm not very familiar with tic/terminfo. As for manpages, I think all that needs to be done is rename `alacritty.man` to `alacritty.1`.

<details>
<summary>Investigating terminfo</summary>

 `tic -x -o /tmp -e alacritty,alacritty-direct alacritty.info` creates `/tmp/61/alacritty` and `/tmp/61/alacritty-direct`. Based on Kitty's [build-terminfo](https://github.com/kovidgoyal/kitty/blob/10435c23c275b20f36bb208728830f15405a2077/build-terminfo#L14-L31), this is what needs to get copied over to `~/.terminfo` or similar.

However, I have no idea what the `61` signifies, nor the `78` in the path of Kitty's, or the `x` that results from using Kitty's `ssh` kitten.

I suppose this is why Kitty just overwrites `$TERMINFO` and points it to `kitty.app/Contents/Frameworks/kitty/terminfo` – it eliminates all of this hassle. I suppose that's a possible future feature.
</summary>

Reply:
> I think all that needs to be done is rename alacritty.man to alacritty.1.

This is incorrect. See our `INSTALL.md` where all this is explained. The `tic` command we need to run can also be found there, though for the output directory we should probably use `target` instead of `/tmp`.

> However, I have no idea what the 61 signifies, nor the 78 in the path of Kitty's, or the x that results from using Kitty's ssh kitten.

You don't really need to know, since that's not relevant. But usually the directory is named after the terminfo file. So the `alacritty` terminfo can be found in `/usr/share/terminfo/a/alacritty`, but `xterm-256color` you would find in `/usr/share/terminfo/x/xterm-256color`.

> I suppose this is why Kitty just overwrites $TERMINFO and points it to kitty.app/Contents/Frameworks/kitty/terminfo – it eliminates all of this hassle.

There really shouldn't be any hassle, this is all quite standard behavior for terminal emulators. Hacking around that will only lead to confusing users that actually know what they're doing, which I see as the primary userbase of Alacritty.

Reply:
Courtesy @Celti:

> ```
> + modify the support for filesystems which do not support mixed-case to
>   generate 2-character (hexadecimal) codes for the lower-level of the
>   filesystem terminfo database (request by Michail Vidiassov).
> ```
> https://invisible-island.net/ncurses/NEWS.html#index-t20071117

Well, this explains a lot.

I think this is also set at compile time, so running tic during the install process makes a lot more sense now.